ui/app/AppLayouts/Chat/ChatColumn.qml

```diff
@@ -0,0 +1,209 @@
+import QtQuick 2.3
+import QtQuick.Controls 1.3
```

Comment:
Do we use both `QtQuick.Controls 1.3` and `2.3`?

Reply:
No idea. Just saw that we have that. Don't know where it comes from

Reply:
Just tested and I can remove it from most places, but apparently SplitLayout comes from 1.3 🤷